src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/tests/Dropdown.test.js

```diff
@@ -96,16 +96,48 @@ test('Click on disabled option will not fire onClick', () => {
         options: [
             {
                 label: 'An option',
-                onClick: clickSpy,
+                onClick: jest.fn(),
```

Comment:
Shouldn't the `clickSpy` be here? Otherwise the check does not work correctly...